keyboards/foxlab/time80/readme.md

```diff
@@ -0,0 +1,27 @@
+# time80
+
+![time80](https://i.imgur.com/8HpYStP.jpg)
```

Comment:
```suggestion
![time80](https://i.imgur.com/8HpYStPl.jpg)
```

Reply:
Smaller is better?

Reply:
Smaller, but also smaller filesize to download.